core/tests/protractor_desktop/coreEditorAndPlayerFeatures.js

```diff
@@ -309,10 +312,50 @@ describe('Core exploration functionality', function() {
       richTextEditor.appendUnorderedList(['an entry', 'another entry']);
     });
     explorationEditorMainTab.setInteraction(
-      'MultipleChoiceInput',
-      [forms.toRichText('option A'), forms.toRichText('option B')]);
+      'MultipleChoiceInput', [{ editAction: 'create'},
+        [forms.toRichText('option A'), forms.toRichText('option B'),
+          forms.toRichText('option C'), forms.toRichText('option D'),
+          forms.toRichText('option E'), forms.toRichText('option F')]]);
+    explorationEditorMainTab.addResponse('MultipleChoiceInput',
+      function(richTextEditor) {
+        richTextEditor.appendBoldText('correct');
+      }, 'final card', true, 'Equals', 'option A');
+    explorationEditorMainTab.getResponseEditor(0)
+      .addRule('MultipleChoiceInput', 'Equals', 'option C');
+    explorationEditorMainTab.getResponseEditor(0)
+      .addRule('MultipleChoiceInput', 'Equals', 'option F');
+    explorationEditorMainTab.getResponseEditor(0)
+      .expectFeedbackInstructionToBe('correct');
+
+    explorationEditorMainTab.getResponseEditor(0).expectMultipleRulesToBe(
+      [['MultipleChoiceInput', 'Equals', ['option A']],
+        ['MultipleChoiceInput', 'Equals', ['option C']],
+        ['MultipleChoiceInput', 'Equals', ['option F']]]);
+    // delete the options corresponding to answer rules.
+    element(by.css('.protractor-test-interaction')).click();
```

Comment:
We like to keep all of the `element(by...))` calls at the top of the file to make the tests more readable. Could you do that here too?

Reply:
done